test/test_lsh.py

```diff
@@ -240,6 +240,45 @@ def test_get_counts(self):
         for table in counts:
             self.assertEqual(sum(table.values()), 2)
 
+    def test_merge(self):
```

Comment:
Can you add test for Redis storage backend too?

Reply:
Okay, will do it.